app/rubin-2025/register-form.jsx

```diff
@@ -79,233 +79,330 @@ class RegisterForm extends React.Component {
+              )}
+            </div>
+
+            <div id="register-form-email-help-message-2">
+              {(!emailInvalidChars && emailInvalidFormat) && (
+                <Translate className="form-help info" content="registerForm.emailInvalidFormat" />
+              )}
+            </div>
+
             {'emailConflict' in this.state.pending && (
               <LoadingIndicator />
             )}
-            {emailConflict && (
-              <span className="form-help error">
-                <Translate content="registerForm.emailConflict" />{' '}
-                <a href={`${window.location.origin}/reset-password`}>
-                  <Translate content="registerForm.forgotPassword" />
-                </a>
-              </span>
-            )}
-            {(this.state.input_email?.length > 0
-              && !emailConflict
-              && !emailInvalidChars
-              && !emailInvalidFormat
-            ) && (
-              <Translate className="form-help success" content="registerForm.looksGood" />
-            )}
-            <Translate className="form-help info right-align" content="registerForm.required" />
-          </span>
+
+            <div id="register-form-email-help-message-3">
+              {emailConflict && (
+                <span className="form-help error">
+                  <Translate content="registerForm.emailConflict" />{' '}
+                  <a href={`${window.location.origin}/reset-password`}>
+                    <Translate content="registerForm.forgotPassword" />
+                  </a>
+                </span>
+              )}
+            </div>
+
+            <div id="register-form-email-help-message-looksGood">
+              {(this.state.input_email?.length > 0
+                && !emailConflict
+                && !emailInvalidChars
+                && !emailInvalidFormat
+                && !('emailConflict' in this.state.pending)
+              ) && (
+                <Translate className="form-help success" content="registerForm.looksGood" />
+              )}
+            </div>
+
+            <Translate
+              className="form-help info right-align"
+              content="registerForm.required"
+              id="register-form-email-help-message-required"
+            />
+          </div>
+
           <input
-            type="text"
-            name="email"
-            id="register-form-email"
-            value={this.state.input_email}
+            autoComplete="email"
+            aria-describedby="register-form-error-message register-form-email-help-message-1 register-form-email-help-message-2 register-form-email-help-message-3 register-form-email-help-message-looksGood register-form-email-help-message-required"
             className="standard-input full"
             disabled={inputDisabled}
+            id="register-form-email"
+            name="email"
             onChange={this.handleUserInput}
+            type="text"
```

Comment:
```suggestion
            type="email"
```

The type probably needs to match the input purpose here. See [Technique H100: Providing properly marked up email and password inputs](https://www.w3.org/WAI/WCAG21/Techniques/html/H100).

This won't make a difference in desktop browsers but should provide the email keyboard in mobile browsers and should announce the field as an email input in JAWS or NVDA.